### PR TITLE
fix: use scan_interval correctly

### PR DESCRIPTION
**Pull request by vinteo**
Fixes: #97

### CODE REVIEW

File: custom_components/opensprinkler/__init__.py [hunk 1]
```diff
@@ -341,7 +340,6 @@ async def async_will_remove_from_hass(self):
         await super().async_will_remove_from_hass()
         self._coordinator.deregister_time_interval_listener()
 
-    @Throttle(SCAN_INTERVAL)
```
Comment (JurajNyiri) [inline custom_components/opensprinkler/__init__.py:344]: @Throttle didn't cause issues in my testing and I use it in my custom components, what is the reason for removal?

File: custom_components/opensprinkler/__init__.py [hunk 2]
```diff
@@ -19,7 +18,7 @@
 from homeassistant.util import slugify, Throttle
 from homeassistant.util.dt import utc_from_timestamp
 
-from .const import DEFAULT_PORT, DOMAIN, SCAN_INTERVAL
+from .const import DEFAULT_PORT, DOMAIN, UPDATE_INTERVAL
```
Comment (JurajNyiri) [inline custom_components/opensprinkler/__init__.py:21]: I would personally leave the original var name SCAN_INTERVAL, it is a standard across hass components

Comment (vinteo) [reply custom_components/opensprinkler/__init__.py:21]: I am not sure how python works but I wanted to keep it in one place so can import it, it is set as `SCAN_INTERVAL` below

File: custom_components/opensprinkler/__init__.py (hunk 1 above)
Comment (vinteo) [reply custom_components/opensprinkler/__init__.py:344]: Throttle from my understanding is to stop it going too fast, for example limiting updates that call API too fast, so if it is < 30 secs it doesn't do anything

Comment (vinteo) [reply custom_components/opensprinkler/__init__.py:344]: changed it slightly

File: custom_components/opensprinkler/__init__.py (hunk 2 above)
Comment (vinteo) [reply custom_components/opensprinkler/__init__.py]: changed it slightly

File: custom_components/opensprinkler/binary_sensor.py [hunk 3]
```diff
@@ -21,9 +21,11 @@
     DOMAIN,
     SERVICE_RUN,
     SERVICE_STOP,
+    UPDATE_INTERVAL,
```
Comment (JurajNyiri) [inline custom_components/opensprinkler/binary_sensor.py:24]: UPDATE_INTERVAL as SCAN_INTERVAL

File: custom_components/opensprinkler/sensor.py [hunk 4]
```diff
@@ -16,9 +16,10 @@
     OpenSprinklerSensor,
     OpenSprinklerStationEntity,
 )
-from .const import CONF_RUN_SECONDS, DOMAIN, SERVICE_RUN, SERVICE_STOP
+from .const import CONF_RUN_SECONDS, DOMAIN, SERVICE_RUN, SERVICE_STOP, UPDATE_INTERVAL
```
Comment (JurajNyiri) [inline custom_components/opensprinkler/sensor.py:19]: UPDATE_INTERVAL as SCAN_INTERVAL

File: custom_components/opensprinkler/switch.py [hunk 5]
```diff
@@ -14,7 +14,9 @@
     OpenSprinklerProgramEntity,
     OpenSprinklerStationEntity,
 )
-from .const import CONF_RUN_SECONDS, DOMAIN, SERVICE_RUN, SERVICE_STOP
+from .const import CONF_RUN_SECONDS, DOMAIN, SERVICE_RUN, SERVICE_STOP, UPDATE_INTERVAL
```
Comment (JurajNyiri) [inline custom_components/opensprinkler/switch.py:17]: UPDATE_INTERVAL as SCAN_INTERVAL

File: custom_components/opensprinkler/__init__.py (hunk 2 above)
Comment (JurajNyiri) [reply custom_components/opensprinkler/__init__.py]: You could import it as SCAN_INTERVAL, that worked in my fix.. if you want to keep name UPDATE_INTERVAL see my recommendation below

Comment (vinteo) [reply custom_components/opensprinkler/__init__.py]: nah it that works I will change it all to SCAN_INTERVAL. Not very experienced with python so bear with me :)

**Comment by vinteo** [comment]
Duplicate of #99

**Pull request closed without merging**